### DISCUSSION TITLE
`WARNING: Plan outline version is outdated. Please update the plan outline.`

### DISCUSSION THREAD

**Discussion by mohamed-atef2022** [argoproj/argo-workflows#13173] [Q&A] [answered]
i have this warrning shown in my argo workflows logs 
here is some info 
1. argo wf version : 3.5.6
2. argo cd version : v2.8.4+c279299
3. kubernetes version : 1.28

**Comment by Joibel** [comment, marked as answer]
Same conversation [in slack](https://cloud-native.slack.com/archives/C01QW9QSSSK/p1718188944465099):

"This doesn't look like an error coming from argo-workflows itself, rather from the pods you are running using argo workflows."

_lightly modified by agilgur5_: add quote here
